include/relic_fp.h

```diff
@@ -362,6 +362,24 @@ typedef rlc_align dig_t fp_st[RLC_FP_DIGS + RLC_PAD(RLC_FP_BYTES)/(RLC_DIG / 8)]
 #define fp_inv(C, A)	fp_inv_lower(C, A)
 #endif
 
+/**
+ * Computes the Legendre symbol of a prime field element. Computes C = (A|P).
+ *
+ * @param[out] C			- the result.
+ * @param[in] A				- the prime field element to compute.
+ */
+#if FP_SMB == BASIC
+#define fp_smb(A)		fp_smb_basic(A)
+#elif FP_SMB == BINAR
+#define fp_smb(A)		fp_smb_binar(A)
+#elif FP_SMB == DIVST
+#define fp_smb(A)		fp_smb_divst(A)
+#elif FP_SMB == JMPDS
+#define fp_smb(A)		fp_smb_divst(A)
```

Comment:
fp_smb_jmpds ?

Reply:
Yup, this and other bugs being fixed right now. :)

Reply:
Done, ready to merge!